src/services/utils.ts

```diff
@@ -73,9 +73,11 @@ export function buildProxy(
 }
 
 export function getEmbed(status: ProxyStatus, proxy: string): APIEmbed {
+  let color: number = Colors.Red
+  if (status.code === 200) color = Colors.Green; else if (status.code > 200 && status.code < 400) color = Colors.Yellow
   return {
     title: `${status.code} for ${proxy} Proxy`,
-    color: { 200: Colors.Green, 500: Colors.Red }[status.code] || Colors.Yellow,
+    color: color,
```

Comment:
Please use the simplified version since the variable name is the same as the key. e.g. `{ color: color }` => `{ color }`

Reply:
Long time no write js. These changes lgtm.